samples/peripheral/802154_phy_test/sample.yaml

```diff
@@ -9,4 +9,22 @@ tests:
       - nrf21540dk_nrf52840
       - nrf5340dk_nrf5340_cpunet
     platform_allow: nrf52840dk_nrf52840 nrf21540dk_nrf52840 nrf5340dk_nrf5340_cpunet
-    tags: ci_build
+    tags: ci_rs_build
+  sample.peripheral.802154_phy_test_auto:
+    build_only: true
+    integration_platforms:
+      - nrf21540dk_nrf52840
+      - nrf52840dk_nrf52840
+    platform_allow: nrf21540dk_nrf52840 nrf52840dk_nrf52840
+    tags: ci_rs_build
+    extra_args: DCONFIG_PTT_ANTENNA_DIVERSITY=y
+      DCONFIG_PTT_ANT_MODE_AUTO=y
+  sample.peripheral.802154_phy_test_manual:
```

Comment:
name should be changed with antenna diversity feature include: `sample.peripheral.802154_phy_test.ant_div_mode_manual`

Reply:
Sample valid only for nrf21540dk platform

Reply:
fixed